mautic/mautic
# Report: Remove filter related class in  aggregators

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:



#### Steps to test this PR:

Small bug in the execution of JS for reports. If you use a boolean type field in agregators it can block operators list for filters.


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2.  Create email stats report
3. Choose any column
4. Add filters for example: Contact Email is not empty
5. Add any GROUP BY
6. Add aggregator COUNT for Read column.
7. Save and see how operators list for email filter is just with equal/not equal expression
8. After PR should work properly
